Plugin.pm

```diff
@@ -1807,6 +1811,7 @@ sub _trackItem {
 	}
 
 	if ( $prefs->get('parentalWarning') && $track->{parental_warning} ) {
+		$item->{title} .= ' [E]';
```

Comment:
Let's really try to leave the title alone. This should be the title and nothing else, as it might be used to look up additional metadata etc.

Reply:
I'd be in support of an extra element for such additions. As you might know I'm often struggling trying to match names and titles to whatever a web site might be using. And all these additions complicate things considerably.